frontend/sw360-portlet/src/main/webapp/html/utils/includes/attachmentsDetail.jsp

```diff
@@ -21,82 +23,155 @@
-            <tr id="attachmentRow1${loop.count}">
-                <td colspan="8" class="attachmentTitle">
-                    <sw360:DisplayDownloadAttachmentFile attachment="${attachment}"
-                                                         contextType="${documentType}"
-                                                         contextId="${documentID}"/>
-                    "<sw360:out value="${attachment.filename}"/>"
-                </td>
-            </tr>
-            <tr id="attachmentRow2${loop.count}" >
-                <td colspan="3" rowspan="2" class="lessPadding">
-                    Type: <sw360:DisplayEnum value="${attachment.attachmentType}"/> <br/>
-                    Status: <sw360:DisplayEnum value="${attachment.checkStatus}"/> <br/>
-                    SHA1-Checksum: <span class="hashvalue">${attachment.sha1}</span>
-                </td>
-                <td>Uploader:</td>
-                <td colspan="4">
-                    <sw360:out value="${attachment.createdBy}"/>,
-                    <sw360:out value="${attachment.createdTeam}"/>,
-                    <sw360:out value="${attachment.createdOn}"/>
-                    <br/>
-                    Comment:
-                    <core_rt:if test="${not empty attachment.createdComment}">
-                        "<sw360:out value="${attachment.createdComment}"/>"
-                    </core_rt:if>
-                    <core_rt:if test="${empty attachment.createdComment}">
-                        -
-                    </core_rt:if>
-                </td>
-            </tr>
-            <tr id="attachmentRow3${loop.count}" >
-                <td>Approver:</td>
-                <td colspan="4">
-                    <core_rt:if test="${not empty attachment.checkedBy}">
-                        <sw360:out value="${attachment.checkedBy}"/>, <sw360:out value="${attachment.checkedTeam}"/>, <sw360:out value="${attachment.checkedOn}"/>
-                        <br/>
-                        Comment:
-                        <core_rt:if test="${not empty attachment.checkedComment}">
-                            "<sw360:out value="${attachment.checkedComment}"/>"
-                        </core_rt:if>
-                        <core_rt:if test="${empty attachment.checkedComment}">
-                            -
-                        </core_rt:if>
-                    </core_rt:if>
-                    <core_rt:if test="${empty attachment.checkedBy}">
-                        Not yet approved.
-                    </core_rt:if>
-                </td>
-            </tr>
-            <tr id="attachmentRow4${loop.count}" class="lastAttachmentRow">
-            </tr>
-        </core_rt:forEach>
-    </core_rt:if>
-
-    <core_rt:if test="${empty attachments}">
-        <tr id="noAttachmentsRow">
-            <td colspan="6">No attachments yet.</td>
-        </tr>
-    </core_rt:if>
-    </tbody>
-</table>
+
+	<core_rt:if test="${empty attachments}">
```

Comment:
There's a lot of mixed indenting by tabs and spaces in the file. Please use 4 spaces for indenting.